src/main/java/com/epam/tc/dto/BoardDto.java

```diff
@@ -0,0 +1,45 @@
+package com.epam.tc.dto;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Accessors(chain = true)
+public class BoardDto {
+
+    @SerializedName("id")
```

Comment:
It looks like @SerializedName can be removed (in all three DTO classes) as it always uses a name which is already equal to a field name.